Tasks/Common/azure-arm-rest/azure-arm-mysql.ts

```diff
@@ -0,0 +1,320 @@
+import msRestAzure = require("./azure-arm-common");
+import webClient = require("./webClient");
+import tl = require('vsts-task-lib/task');
+import util = require("util");
+import azureServiceClient = require("./AzureServiceClient");
+import Q = require("q");
+
+export class AzureMysqlManagementClient extends azureServiceClient.ServiceClient {
+    public firewallRules: FirewallRules;
+    public mysqlServers: MysqlServers;
+
+    constructor(credentials: msRestAzure.ApplicationTokenCredentials, subscriptionId, baseUri?: any, options?: any) {
+        super(credentials, subscriptionId);
+
+        this.apiVersion = '2017-04-30-preview';
+        this.acceptLanguage = 'en-US';
```

Comment:
is this default?

Reply:
In API Doc they have not specified but this is same pattern we are following in other file for network call.